android/build.gradle

```diff
@@ -3,6 +3,7 @@
 buildscript {
     ext {
         buildToolsVersion = "33.0.0"
+        kotlin_version = '1.8.0'
```

Comment:
Does it matter if we use `1.9.10`? :)

Reply:
I got the version number from here:
https://github.com/AtB-AS/mittatb-app/blob/13d7e7f05292966e4faf28b4ebdd63fbe3bcb820/android/app/build.gradle#L141-L142
I think this is correct?
Or do you mean if we should upgrade?